front_end/panels/ai_chat/agent_framework/AgentRunner.ts

```
@@ -666,6 +701,7 @@ export class AgentRunner {
           systemPrompt: currentSystemPrompt,
           tools: toolSchemas,
           temperature: temperature ?? 0,
+          agentName: agentName,  // Pass agent identity for provider-specific routing
```

ESLint

[error] Expected property shorthand.

(object-shorthand)